aws_jobs/cyhy-data-extract.py

```diff
@@ -285,22 +309,22 @@ def main():
         'findings': {}
     }
 
-    if args['--cyhy_section']:
+    if args['--cyhy-config']:
         for collection in cyhy_collection:
             query_data(cyhy_db[collection], cyhy_collection[collection],
                        tbz_file, tbz_filename, end_of_data_collection)
-    if args['--scan_section']:
+    if args['--scan-config']:
         for collection in scan_collection:
             query_data(scan_db[collection], scan_collection[collection],
                        tbz_file, tbz_filename, end_of_data_collection)
-    if args['--assessment_section']:
+    if args['--assessment-config']:
         for collection in assessment_collection:
             query_data(assessment_db[collection],
                        assessment_collection[collection],
                        tbz_file, tbz_filename, end_of_data_collection)
 
     # Note that we use the elasticsearch AWS profile here
-    json_data = util.to_json(get_dmarc_data(ES_REGION, ES_URL,
+    json_data = to_json(get_dmarc_data(ES_REGION, ES_URL,
```

Comment:
The lines that immediately follow are no longer indented properly.  Please fix that.